app/views/account/companies/html/_social_links.html.slim

```diff
@@ -1,8 +1,9 @@
 - social_list = %w[twitter facebook linkedincompany youtube angellist owler crunchbasecompany pinterest google klout]
```

Comment:
never do like this
add this as constant in `Company` model
and make it work from company instance method
```
class Company
  SOCIAL_LINKS = %w[ twitter ... ].freeze

  def socials
    SOCIAL_LINKS.map { |social| self[social] }.compact
  end
end
```

and in view

```
- company.socials.each do |social_link|
 = render partial: 'copmany/socials', social_link: social_link
```